### PR TITLE
feat: add striking stardust

### PR DESCRIPTION
**Pull request by giantspacemonster14**
## Please make sure to check the following tasks before opening and submitting a PR

* [X] I understand and have followed the [contribution guide](https://github.com/revoltchat/revolt/discussions/282)
* [X] I have tested my changes locally and they are working as intended
* [X] These changes do not have any notable side effects on other Revolt projects

sorry if any mistakes were made

### CODE REVIEW

File: data/striking-stardust/Preset.toml [hunk 1]
```diff
@@ -0,0 +1,44 @@
+slug = "striking-stardust"
+name = "Striking Stardust"
+creator = "giantspacemonster"
+description = "a vibrant purple theme"
+tags = ["purple", "dark"]
+‎
```
Comment (Rexogamer) [inline data/striking-stardust/Preset.toml:6]:
```suggestion

```

File: data/striking-stardust/Preset.toml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+name = "Striking Stardust"
+creator = "giantspacemonster"
+description = "a vibrant purple theme"
+tags = ["purple", "dark"]
+[variables]
+light = false
+accent = "#00b46c"
+background = "#160a33"
+foreground = "#9dffec"
+‎
```
Comment (Rexogamer) [inline data/striking-stardust/Preset.toml:11]:
```suggestion

```

File: data/striking-stardust/Preset.toml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+block = "#2e1948"
+message-box = "#461e59"
+mention = "11393c"
+‎
```
Comment (Rexogamer) [inline data/striking-stardust/Preset.toml:15]:
```suggestion

```

File: data/striking-stardust/Preset.toml [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+success = "#65E572"
+warning = "#FAA352"
+error = "#F06464"
+hover = "rgba(0, 0, 0, 0.1)"
+‎
```
Comment (Rexogamer) [inline data/striking-stardust/Preset.toml:20]:
```suggestion

```

File: data/striking-stardust/Preset.toml [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+[variables.scrollbar]
+thumb = "#00b46c"
+track = "transparent"
+‎
```
Comment (Rexogamer) [inline data/striking-stardust/Preset.toml:24]:
```suggestion

```

File: data/striking-stardust/Preset.toml [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+[variables.primary]
+background = "#241440"
+header = "#461e59"
+‎
```
Comment (Rexogamer) [inline data/striking-stardust/Preset.toml:28]:
```suggestion

```

File: data/striking-stardust/Preset.toml [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+[variables.secondary]
+background = "#2e1948"
+foreground = "#51dff4"
+header = "#3c1c55"
+‎
```
Comment (Rexogamer) [inline data/striking-stardust/Preset.toml:33]:
```suggestion

```

File: data/striking-stardust/Preset.toml [hunk 8]
```diff
@@ -0,0 +1,43 @@
...
+[variables.tertiary]
+background = "#3c1c55"
+foreground = "#44bcfa"
+‎
```
Comment (Rexogamer) [inline data/striking-stardust/Preset.toml:37]:
```suggestion

```

**Review by Rexogamer — CHANGES_REQUESTED** [review]
Fixing LTR marks; just out of curiosity, what text editor are you using?

**Review by Rexogamer — APPROVED** [review]

**Comment by giantspacemonster14** [comment]
> Fixing LTR marks; just out of curiosity, what text editor are you using?
It was an app on my iOS device, it might have made some autocorrect issues, or maybe it’s just the app

**Pull request merged**